models/article-model.js

```diff
@@ -27,14 +26,18 @@ FROM
     articles
 LEFT JOIN 
     comments ON articles.article_id = comments.article_id
-GROUP BY 
-    articles.article_id
-ORDER BY 
-    articles.created_at DESC`
-      )
-      .then(({ rows }) => rows);
-    )
-    .then(({ rows }) => rows);
+    `;
+  const queryValues = [];
+
+  if (topic) {
+    queryValues.push(topic);
+    query += ` WHERE topic = $1`;
+  }
+
+  query += `GROUP BY articles.article_id 

```

Comment:
Good job incorporating this into your pre-existing selectArticles rather than creating a new model! :)